src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private double firstLeg;
+
+    public Square(String color, double firstLeg) {
```

Comment:
According to the checklist, don't put all behavior into a single interface if the methods are conceptually different from each other. In this case, the `calculateArea()` method should be in a separate interface, as it is not conceptually similar to the `drawFigure()` method.